cmd/snap-bootstrap/initramfs_systemd_mount.go

```diff
@@ -144,48 +143,45 @@ func doSystemdMountImpl(what, where string, opts *systemdMountOptions) error {
 		args = append(args, "--options="+strings.Join(options, ","))
 	}
 
-	// note that we do not currently parse any output from systemd-mount, but if
-	// we ever do, take special care surrounding the debug output that systemd
-	// outputs with the "debug" kernel command line present (or equivalently the
-	// SYSTEMD_LOG_LEVEL=debug env var) which will add lots of additional output
-	// to stderr from systemd commands
-	out, err := exec.Command("systemd-mount", args...).CombinedOutput()
-	if err != nil {
-		return osutil.OutputErr(out, err)
-	}
-
 	// if it should survive pivot_root() then we need to add overrides for this
 	// unit to /run/systemd units
 	if !opts.Ephemeral {
-		// to survive the pivot_root, we need to make the mount units depend on
-		// all of the various initrd special targets by adding runtime conf
-		// files there
+		args = append(args, "--property=DefaultDependencies=no")
```

Comment:
This could use a little comment explaining why DefaultDependencies=no

Reply:
I suppose this was explained in the other core-initrd PRs which is why I didn't think twice about it, but yes I agree a comment here specifically would be good

Reply:
I commented it.